src/doc/rustc/src/platform-support.md

```diff
@@ -169,6 +169,7 @@ target | std | notes
 `i686-unknown-freebsd` | ✓ | 32-bit FreeBSD [^x86_32-floats-return-ABI]
 `i686-unknown-linux-musl` | ✓ | 32-bit Linux with musl 1.2.3 [^x86_32-floats-return-ABI]
 [`i686-unknown-uefi`](platform-support/unknown-uefi.md) | * | 32-bit UEFI
+[`loongarch64-unknown-linux-musl`](platform-support/loongarch-linux.md) | ? |  | LoongArch64 Linux (LP64D ABI) with musl 1.2.3
```

Comment:
Turns out this was added in the tier 2 section, not the tier 3 section. That's pretty hard to see during review since the diff doesn't show the section header...

I wonder if there's something we can do here, like split the tables across multiple files?